package.json

```diff
@@ -80,20 +80,25 @@
   },
   "dependencies": {
     "@date-io/moment": "^1.3.13",
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
```

Comment:
Should be moved to `peerDependencies` + `devDependencies`, same as for material-ui and atdesign.

If user wants to use vailla widgets, he don't need to another frameworks dependencies to be installed.
See in readme: 
```
For AntDesign widgets only:
npm i antd --save
```
Same should be noted for bootstrap.

In future I want to decompose lib to @rawb/core, @raqb/antd etc. with workspaces

Reply:
@ukrbublik Done - I also moved bootstrap css import to the newly-created bootstrap's index.js file.
Thanks